decidim-admin/spec/commands/activate_participatory_process_step_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "spec_helper"
+
+describe Decidim::Admin::DeactivateParticipatoryProcessStep do
+  let(:process_step) { create :participatory_process_step, :active }
+
+  before do
+    @success = nil
+    @failure = nil
+  end
+
+  def success
+    @success = true
+  end
+
+  def failure
+    @failure = true
+  end
+
+  subject do
+    described_class.call(process_step) do
+      on(:ok) { success }
+      on(:invalid) { failure }
+    end
+  end
+
+  context "when the step is nil" do
+    let(:process_step) { nil }
+
+    it "is not valid" do
+      subject
+      expect(@failure).to eq true
```

Comment:
You can do this and get rid of all the boilerplate:

`expect { command.call }.to broadcast(:ok)`